src/authentication/authenticationContext.ts

```diff
@@ -64,9 +64,9 @@ export class AuthenticationContext {
         return promise;
     }
 
-    public async signIn(): Promise<void> {
+    public async signIn(options: object): Promise<void> {
```

Comment:
This should be an optional parameter. Could you also add a type definition for the object?